### PR TITLE
Electron packaging

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -50,7 +71,7 @@ function startQtumProcess(reindex) {
     flags.push('-reindex');
   }
 
-  const qtumProcess = spawn(qtumdPath, flags);
+  qtumProcess = spawn(qtumdPath, flags, { detached: true });
```
Comment (frankobe) [inline src/index.js]: in detached mode?
```
On Windows, setting options.detached to true makes it possible for the child process to continue running after the parent exits. The child will have its own console window. Once enabled for a child process, it cannot be disabled.

On non-Windows platforms, if options.detached is set to true, the child process will be made the leader of a new process group and session. Note that child processes may continue running after the parent exits regardless of whether they are detached or not.
```

File: src/index.js [hunk 2]
```diff
@@ -80,6 +101,9 @@ function startQtumProcess(reindex) {
   qtumProcess.on('close', (code) => {
     logger.debug(`qtumd exited with code ${code}`);
   });
+
+  // repeatedly check if qtumd is running
+  checkInterval = setInterval(checkQtumd, 1000);
```
Comment (frankobe) [inline src/index.js]: will `setTimeout` be easier? No need to cleanInterval

File: src/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/index.js]: I believe I had some issues with running it in attached mode. I will test again.

File: src/index.js (hunk 2 above)
Comment (dwalintukan) [reply src/index.js]: `setInterval` is for repeating calls. Wanted to make sure qtumd is running fully before launching all the services.

File: src/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/index.js]: Just tested, looks ok to run in attached mode. Must have fixed it forgot to remove the flag.

File: src/index.js (hunk 2 above)
Comment (frankobe) [reply src/index.js]: @dwalintukan this line is a recursive call to start itself so dunt need repeating calls.

Comment (dwalintukan) [reply src/index.js]: It's not recursive. The `checkInterval` is holding the interval timer itself and `checkQtumd` only has logic to clear the interval.

**Pull request merged**